business_requirement_activity_report/README.rst

```diff
@@ -2,9 +2,9 @@
    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
    :alt: License: AGPL-3
 
-=========================
-Timesheet Activity Report
-=========================
+====================================
+Business Requirement Activity Report
```

Comment:
> To replace #109 Please have a check
Please have a check @noahzaozao @elicoidal @seb-elico

Not really idea I continue the left PR created @xie8899 to solve merge problems and not sure about this maybe @elicoidal could reply or you could have a look on previous history of comments.